api/proto/dsse.proto

```diff
@@ -16,16 +16,26 @@ syntax = "proto3";
 package dev.sigstore.rekor.v2;
 
 import "google/api/field_behavior.proto";
+import "sigstore_common.proto";
+import "envelope.proto";
 
 import "verifier.proto";
 
 option go_package = "github.com/sigstore/rekor-tiles/pkg/generated/protobuf";
 
 // A request to add a dsse entry to the log
-message DSSERequest {
-    // A DSSE envelope (as defined by https://github.com/secure-systems-lab/dsse/blob/master/envelope.proto)
-    // in stringified JSON
-    string envelope = 1 [(google.api.field_behavior) = REQUIRED];
-    // collection of verification material used to verify
-    repeated Verifier verifier = 2;
+message DSSERequestV0_0_2 {
+    // A DSSE envelope
+    io.intoto.Envelope envelope = 1 [(google.api.field_behavior) = REQUIRED];
+    // Signatures and their associated verification material used to verify the payload in the envelope
+    // These should be a direct 1:1 mapping with the signatures provided in the envelope
```

Comment:
Would we err out if there's an unequal number of verifiers and signatures? If not, I might update wording to say that all signatures must be verified using verifiers (so that additional verifiers could be specified and that's not an error)